### PR TITLE
Initial contribution of the Helios IP (www.2n.cz) binding

### PR DESCRIPTION
**Pull request by kgoderis**
Signed-off-by: Karel Goderis <karel.goderis@me.com>

### CODE REVIEW

**Comment by kgoderis** [comment]
@martinvw Jenkins is failing with

22:13:39.903  |  Failed to transfer Could not find metadata org.openhab:pom-tycho:2.2.0-SNAPSHOT/maven-metadata.xml in cloudbees-private-snapshot-repository (https://repository-openhab.forge.cloudbees.com/snapshot)
22:13:39.903  |  Downloaded artifact https://openhab.jfrog.io/openhab/libs-snapshot/org/openhab/pom-tycho/2.2.0-SNAPSHOT/maven-metadata.xml
22:13:40.034  |  Failed to transfer Could not find metadata org.openhab:pom:2.2.0-SNAPSHOT/maven-metadata.xml in cloudbees-private-snapshot-repository (https://repository-openhab.forge.cloudbees.com/snapshot)
22:13:40.095  |  Downloaded artifact https://openhab.jfrog.io/openhab/libs-snapshot/org/openhab/pom/2.2.0-SNAPSHOT/maven-metadata.xml
22:13:40.459  |  Failed to transfer Could not find metadata org.openhab.binding:pom:2.1.0-SNAPSHOT/maven-metadata.xml in cloudbees-private-snapshot-repository (https://repository-openhab.forge.cloudbees.com/snapshot)
22:13:40.477  |  Failed to transfer Could not find artifact org.openhab.binding:pom:pom:2.1.0-SNAPSHOT in cloudbees-private-snapshot-repository (https://repository-openhab.forge.cloudbees.com/snapshot)
22:13:42.200  |  ERROR: Failed to parse POMs

File: addons/binding/org.openhab.binding.helios/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Helios Binding
+Bundle-SymbolicName: org.openhab.binding.helios;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.helios/META-INF/MANIFEST.MF]: Make sure this has the correct reference

File: addons/binding/org.openhab.binding.helios/pom.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.binding</groupId>
+    <artifactId>pom</artifactId>
+    <version>2.1.0-SNAPSHOT</version>
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.helios/pom.xml]: Update to the correct version

File: addons/binding/org.openhab.binding.helios/pom.xml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  </parent>
+
+  <properties>
+    <bundle.symbolicName>org.openhab.binding.helios</bundle.symbolicName>
+    <bundle.namespace>org.openhab.binding.helios</bundle.namespace>
+  </properties>
+
+  <groupId>org.openhab.binding</groupId>
+  <artifactId>org.openhab.binding.helios</artifactId>
+  <version>2.1.0-SNAPSHOT</version>
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.helios/pom.xml]: Also

File: addons/binding/org.openhab.binding.helios/src/main/java/org/openhab/binding/helios/internal/ws/soap/SOAPSubscriptionActionHandler.java [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+/**
+ * The {@link SOAPSubscriptionActionHandler} is a custom SOAP handler that modifies
+ * some SOAP headers in order to get the Helios comm. working
+ *
+ * @author Karel Goderis - Initial contribution
+ */
+public class SOAPSubscriptionActionHandler extends SOAPActionHandler {
+
+    private Logger logger = LoggerFactory.getLogger(HeliosHandler27.class);
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.helios/src/main/java/org/openhab/binding/helios/internal/ws/soap/SOAPSubscriptionActionHandler.java]: Wrong logger class

File: addons/binding/org.openhab.binding.helios/META-INF/MANIFEST.MF [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+ org.eclipse.smarthome.core.scheduler,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding,
+ org.eclipse.smarthome.core.thing.binding.builder,
+ org.eclipse.smarthome.core.thing.type,
+ org.eclipse.smarthome.core.types,
+ org.slf4j,
+ org.eclipse.jdt.annotation;resolution:=optional
+Service-Component: OSGI-INF/*.xml
+Export-Package: org.openhab.binding.helios,
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.helios/META-INF/MANIFEST.MF]: You should also import all packages which you export, see [B.8](http://docs.openhab.org/developers/development/guidelines.html#b-osgi-bundles)

**Pull request closed without merging**